### PR TITLE
Updated Static Data V3

### PR DESCRIPTION
**Pull request by JanOuborny**
Ref: https://discussion.developer.riotgames.com/articles/1736/updates-to-lol-static-data-v3.html

### CODE REVIEW

File: RiotSharp/StaticDataEndpoint/StatsStatic.cs [hunk 1]
```diff
@@ -75,100 +75,100 @@ public class StatsStatic
 
         public double PercentSpellVampMod { get; set; }
 
-        [JsonProperty("rFlatArmorModPerLevel")]
-        public double RFlatArmorModPerLevel { get; set; }
+        [JsonProperty("FlatArmorModPerLevel")]
+        public double FlatArmorModPerLevel { get; set; }
```
Comment (BenFradet) [inline RiotSharp/StaticDataEndpoint/StatsStatic.cs]: The property name seems to match `JsonProperty`, can't we get rid of the `JsonProperty`?

File: RiotSharp/StaticRiotApi.cs [hunk 2]
```diff
@@ -888,7 +895,7 @@ public RealmStatic GetRealm(Region region)
                     string.Format("locale={0}", language.ToString()),
                     summonerSpellData == SummonerSpellData.basic ?
                         string.Empty :
-                        string.Format("spellData={0}", summonerSpellData.ToString())
+                        string.Format("tags={0}", summonerSpellData.ToString())
```
Comment (BenFradet) [inline RiotSharp/StaticRiotApi.cs]: might be worth it to create a variable `tags={0}`, don't you think?

File: RiotSharp/StaticDataEndpoint/StatsStatic.cs (hunk 1 above)
Comment (JanOuborny) [inline RiotSharp/StaticDataEndpoint/StatsStatic.cs]: Yes, we can 👍

File: RiotSharp/StaticRiotApi.cs (hunk 2 above)
Comment (JanOuborny) [inline RiotSharp/StaticRiotApi.cs]: Seems logical. I will do that.

**Pull request merged**